docs/cookbook/disabling-localised-urls.rst

```diff
@@ -0,0 +1,52 @@
+How to disable localised URLs?
+==============================
+
```

Comment:
Don't know, they just don't want to have localised URLs, I'm bad at creating content 🌮